packages/framework/collabspaces/src/test/collabSpaces.spec.ts

```diff
@@ -60,11 +61,20 @@ describe("Temporal Collab Spaces", () => {
 	let loader: IHostLoader | undefined;
 	let seed: number;
 
+	const summaryOptionsToDisableHeuristics: ISummaryRuntimeOptions = {
+		summaryConfigOverrides: {
+			state: "disableHeuristics",
```

Comment:
Great call out RE default summarizer! I bet that's what was holding MSNs!
I'm curious if perf improves simply because MSN can move more freely.
If it is, it would be an interesting option to keep to demonstrate perf problems.